src/plugins/usage_collection/server/routes/integration_tests/stats.test.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { BehaviorSubject } from 'rxjs';
+import { UnwrapPromise } from '@kbn/utility-types';
+
+import {
+  AuthStatus,
+  MetricsServiceSetup,
+  ServiceStatus,
+  ServiceStatusLevels,
+} from '../../../../../core/server';
+import {
+  contextServiceMock,
+  loggingSystemMock,
+  metricsServiceMock,
+} from '../../../../../core/server/mocks';
+import { createHttpServer } from '../../../../../core/server/test_utils';
+import { registerStatsRoute } from '../stats';
+import supertest from 'supertest';
+import { CollectorSet } from '../../collector';
+
+type HttpService = ReturnType<typeof createHttpServer>;
+type HttpSetup = UnwrapPromise<ReturnType<HttpService['setup']>>;
+
+describe('/api/stats', () => {
```

Comment:
The test coverage on this endpoint are admittedly low. Since this will be owned by @elastic/telemetry and they have more knowledge of usage collectors (which is the bulk of the logic in this endpoint), I'm leaving this to them to flesh out more. This simply serves as a smoke test to ensure the request will complete successfully. There are also existing integration tests that test more permutations of auth & query parameter combos.

Reply:
> Since this will be owned by @elastic/telemetry 

Are you going to update CODEOWNERS in the following PR?

Reply:
They already own this entire plugin, so I don't believe any changes are needed to CODEOWNERS?